packages/react-native-reanimated/src/common/types/style.ts

```diff
@@ -32,3 +32,25 @@ export type TransformsArray = Exclude<
   TransformsStyle['transform'],
   string | undefined
 >;
+
+export type ParsedDropShadow = {
+  offsetX: number;
+  offsetY: number;
+  standardDeviation: number;
+  color: number;
+};
+
+export type ParsedFilterFunction = {
+  brightness?: number;
+  contrast?: number;
+  dropShadow?: ParsedDropShadow;
+  grayscale?: number;
+  hueRotate?: number;
+  invert?: number;
+  opacity?: number;
+  saturate?: number;
+  sepia?: number;
+  blur?: number;
+};
```

Comment:
```suggestion
export type ParsedFilterFunction =
  | { brightness?: number }
  | { contrast?: number }
  | { dropShadow?: ParsedDropShadow }
  | { grayscale?: number }
  | { hueRotate?: number }
  | { invert?: number }
  | { opacity?: number }
  | { saturate?: number }
  | { sepia?: number }
  | { blur?: number };
```

Won't this type be more appropriate?